source/common/ssl/context_config_impl.cc

```diff
@@ -25,10 +25,13 @@ const std::string ContextConfigImpl::DEFAULT_CIPHER_SUITES =
     "AES256-SHA256:"
     "AES256-SHA";
 
+const std::string ContextConfigImpl::DEFAULT_ECDH_CURVES = "X25519:P-256";
```

Comment:
@PiotrSikora I don't really have an opinion. TBH I'm not qualified to review TLS changes like this. Can you find one (optimally 2) Google people to review? I can review for code style, etc. but that's about it.